config/config.yaml

```diff
@@ -9,23 +9,19 @@ results_dir: "results/"
 ABC_DIR_PATH: "ABC"
 
 ### Reference Files
-gene_TSS500: "resources/RefSeqCurated.170308.bed.CollapsedGeneBounds.hg38.TSS500bp.bed"
-chr_sizes: "resources/GRCh38_EBV.no_alt.chrom.sizes.tsv"
-ubiq_expr_genes: "resources/for_classifier_use_ubiquitous_expression_RefSeqCurated.170308.bed.CollapsedGeneBounds.hg38.TSS500bp.txt"
+gene_TSS500: "reference/RefSeqCurated.170308.bed.CollapsedGeneBounds.hg38.TSS500bp.bed"
+chr_sizes: "reference/GRCh38_EBV.no_alt.chrom.sizes.tsv"
+ubiq_expr_genes: "reference/for_classifier_use_ubiquitous_expression_RefSeqCurated.170308.bed.CollapsedGeneBounds.hg38.TSS500bp.txt"
 
 # Filling new features into prediction file
-feature_config: "config/feature_config.tsv"
-crispr_dataset: "resources/EPCrisprBenchmark_ensemble_data_GRCh38.tsv.gz"
+crispr_dataset: "reference/EPCrisprBenchmark_ensemble_data_GRCh38.tsv.gz"
 
 # ENCODE E2G
-feature_table:
-  ATAC: "/oak/stanford/groups/engreitz/Users/sheth/ENCODE-E2G_v2/data/feature_table/model_scATAC_IGVF.tsv"
-  DHS: "/oak/stanford/groups/engreitz/Users/atan5133/ENCODE-E2G/data/feature_table/model_ablation_encode_e2g.tsv"
-models_dir:
-  ATAC: "/oak/stanford/groups/engreitz/Users/sheth/ENCODE-E2G_v2/models_scATAC_IGVF_filt/K562_cellranger_1/"
-  DHS: "/oak/stanford/groups/engreitz/Users/atan5133/ENCODE-E2G/models"
+MEGAMAP_HIC_FILE: https://s3.us-central-1.wasabisys.com/aiden-encode-hic-mirror/bifocals_iter2/tissues.hic
```

Comment:
why is this needed here? shouldn't this be part of the ABC module?

Reply:
We use this to determine to use the megamap model. We compare the hic_file in the biosamples config. See `_get_biosample_model_dir` fn in utils.smk